Packs/fireeye/Integrations/integration-fireeye.yml

```diff
@@ -362,6 +362,7 @@ script:
                 if (contentType && contentType.indexOf('application/xml') !== -1) {
                   status_result = JSON.parse(x2j(response.Body));
                 }
+                status_result['submission_Key'] = submissionKeyList[i];
```

Comment:
need to handle the `fe-submit-url-status` also in these case, I believe it is lacking.

Reply:
I don't think so. As I mentioned above, the fix here is due to a change in behavior I previously did to `fe-submit-status` **alone** (to handle multiple submissions at once). So unless we have a way to reproduce this same issue we had with `fe-submit-status` when trying to submit multiple ids in the `fe-submit-url-status` this is not needed. (due to the same if statement you commented above )